spec/lib/importers/licence_transaction/licence_importer_spec.rb

```diff
@@ -134,6 +136,8 @@ def expected_put_content_payload
           licence_transaction_continuation_link: "http://www.hpc-uk.org/apply",
           licence_transaction_licence_identifier: licence_identifier,
           licence_transaction_will_continue_on: "the Health and Care Professions Council (HCPC) website",
+          licence_transaction_industry: %w[healthcare],
+          licence_transaction_location: %w[england wales scotland northern-ireland],
```

Comment:
Why have these rows been added here? Was this just missed in the original tests? I assume so, as I assume that we were always sending the industry and location information in the metadata.

Reply:
Previously, we were stubbing out the licences-api response as an empty array and so there were no industries / locations in the test examples (https://github.com/alphagov/specialist-publisher/blob/e7169d721eeef6ab5057a617d0f7834ecb52321f/spec/lib/importers/licence_transaction/licence_importer_spec.rb#L18), now we’re using the licences_and_tagging.csv file which defines industries and locations.